### PR TITLE
Add empty imput field to protect against bots

### PR DESCRIPTION
**Pull request by garyanewsome**
###### Add empty input field to protect against bots

- Added empty field to index.html
- add check/guard in `mailer.rb` to verify the field is `empty?`, if it is not, redirect to `err_bot.haml`
- added test

### CODE REVIEW

File: tests/test.rb [hunk 1]
```diff
@@ -44,11 +44,23 @@ def test_mailer_redirects_to_error_without_valid_configured_account
...
+  def test_mailer_redirects_to_err_bot_if_verify_field_is_not_empty
+    post '/', { public_token: 'default',
+                name: 'butt',
+                email: 'butt@tp.com',
+                verify: 'ima bot bot',
+                action: 'value' }
+
+    assert last_response.redirect?
+    follow_redirect!
+    assert_includes last_response.body, 'ERR! botnope'
```
Comment (ninjapanzer) [inline tests/test.rb]: we don't actually want to return a success or fail response. We don't want the bot or someone looking at the logs to know that we blocked them

Comment (garyanewsome) [reply tests/test.rb]: I changed it to redirect to root/index

**Comment by garyanewsome** [comment]
@ninjapanzer tag your it

**Comment by ninjapanzer** [comment]
@garyanewsome Tests are now passing

**Comment by garyanewsome** [comment]
So merge? Do we deploy? Or ...

**Pull request merged**